cli/run_util.go

```diff
@@ -706,22 +706,14 @@ func activateAndRunSteps(
 		// Run step
 		bitrise.PrintRunningStepHeader(stepInfoPtr, mergedStep, idx)
 		if mergedStep.RunIf != nil && *mergedStep.RunIf != "" {
-			outStr, err := tools.EnvmanJSONPrint(configs.InputEnvstorePath)
+			envList, err := tools.EnvmanJSONPrint(configs.InputEnvstorePath)
 			if err != nil {
 				runResultCollector.registerStepRunResults(&buildRunResults, stepStartTime, mergedStep, stepInfoPtr, stepIdxPtr,
 					*mergedStep.RunIf, models.StepRunStatusCodePreparationFailed, 1, fmt.Errorf("EnvmanJSONPrint failed, err: %s", err),
 					isLastStep, false, map[string]string{}, stepStartedProperties)
 				continue
 			}
 
-			envList, err := envmanModels.NewEnvJSONList(outStr)
-			if err != nil {
-				runResultCollector.registerStepRunResults(&buildRunResults, stepStartTime, mergedStep, stepInfoPtr, stepIdxPtr,
-					*mergedStep.RunIf, models.StepRunStatusCodePreparationFailed, 1, fmt.Errorf("CreateFromJSON failed, err: %s", err),
-					isLastStep, false, map[string]string{}, stepStartedProperties)
-				continue
-			}
-
```

Comment:
It is probably obvious but I do not see from the PR why this was removed. Can you remind me why?

Reply:
Updated the function names here: https://github.com/bitrise-io/bitrise/pull/819/commits/14e000e7617f23871bf5859d3328e64ee2f94b39

- `EnvmanInitAtPath` + (old) `EnvmanInit` -> `EnvmanInit`
- `EnvmanAdd`
- `ExportEnvironmentsList` -> `EnvmanAddEnvs`
- `EnvmanJSONPrint` -> `EnvmanReadEnvList`

With this, the old `tools.EnvmanJSONPrint` + `envmanModels.NewEnvJSONList` became `tools.EnvmanReadEnvList`